api/request_source/request_source_plugin.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package nighthawk.request_source;
+
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+import "api/client/options.proto";
+
+// Configuration for DummyPluginRequestSource (plugin name: "nighthawk.dummy-request-source-plugin")
+// that does nothing
+message DummyPluginRequestSourceConfig {
```

Comment:
These names are fairly repetitive. They all currently live in the request_source namespace, but also contain the words "RequestSource". Can we remove one or the other?

This especially shows in the name RequestOptionsListPluginRequestSourceConfig which reached a length that makes it hard to parse meaning from it.